packages/addresses/src/curve.ts

```diff
@@ -33,10 +33,11 @@ export type OffCurveAddress<TAddress extends string = string> = Brand<Address<TA
 export function isOffCurveAddress(
     putativeOffCurveAddress: string,
 ): putativeOffCurveAddress is OffCurveAddress<typeof putativeOffCurveAddress> {
-    if (!isAddress(putativeOffCurveAddress)) return false;
```

Comment:
I generally avoid this style, because when you…

1. Add a statement to the `if` block
2. Add an else block

The diff in the PR ends up being 5 lines instead of 1.

Compare:

```diff
- if (!isAddress(putativeOffCurveAddress)) return false;
+ if (!isAddress(putativeOffCurveAddress)) {
+     // <- PR adds this line
+     return false;
+ }
```

To this:
```diff
  if (!isAddress(putativeOffCurveAddress)) {
+     // <- PR adds this line
      return false;
  }
```